src/Pelagos/Util/DOIutil.php

```diff
@@ -319,42 +319,47 @@ public function getDOIMetadata($doi)
      * @throws HttpClientErrorException When there was an 4xx error negotiating with EZID.
      * @throws HttpServerErrorException When there was an 5xx error negotiating with EZID.
      *
-     * @return array
+     * @return boolean
```

Comment:
Always returns true if successful, or throws exception, might as well return void?
Because you would have to call this in a try, checking the return value wouldn't help you much.
It's just a thought, I don't know what the best to handle this is, just my opinion.

Reply:
That is right, there is no significance of the return value from this method, and even in the Consumer it doesn't expect a return value when this method is called